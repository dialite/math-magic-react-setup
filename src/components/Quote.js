import React from 'react';
import '../App.css';
import style from './styles.module.css';

const Quote = () => (
  <div className="quoteContainer">
    <div className="quote">
      <p className={style.marginLeftAndRight}>
        Mathematics is not about numbers, equations, or algorithms: it is about
        understanding. —William Paul Thurston
      </p>
      <p className={style.marginLeftAndRight}>
        Pure mathematics is, in its way, the poetry of logical ideas. — Albert
        Einstein, German theoretical physicist
      </p>
      <p className={style.marginLeftAndRight}>
        Without mathematics, there’s nothing you can do. Everything around you
        is mathematics. Everything around you is numbers. — Shakuntala Devi,
        Indian writer and mental calculator
      </p>
      <p className={style.marginLeftAndRight}>
        Mathematics is the most beautiful and most powerful creation of the
        human spirit. — Stefan Banach, Polish mathematician
      </p>
      <p className={style.marginLeftAndRight}>
        We will always have STEM with us. Some things will drop out of the
        public eye and go away, but there will always be science, engineering,
        and technology. And there will always, always be mathematics. —
        Katherine Johnson, African-American mathematician
      </p>
      <p className={style.marginLeftAndRight}>
        Mathematics as an expression of the human mind reflects the active will,
        the contemplative reason, and the desire for aesthetic perfection. Its
        basic elements are logic and intuition, analysis and construction,
        generality and individuality. — Richard Courant, German-American
        mathematician
      </p>
    </div>
  </div>
);

export default Quote;
