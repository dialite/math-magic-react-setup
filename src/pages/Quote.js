import '../modules-css/quote.css';

const Quote = () => (
  <div className="main-container-quote">
    <div className="quote-container">
      <span className="span">
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding.
        <hr />
        <strong className="strong">- William Paul Thurston</strong>
      </span>
    </div>
  </div>
);

export default Quote;
