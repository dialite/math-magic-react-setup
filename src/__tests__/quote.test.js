import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Render Quote', () => {
  const renderQuote = renderer.create(<Quote />).toJSON();
  expect(renderQuote).toMatchSnapshot();
});
