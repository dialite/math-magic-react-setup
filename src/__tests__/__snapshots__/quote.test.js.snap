// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Quote 1`] = `
<div
  className="quoteContainer"
>
  <div
    className="quote"
  >
    <p
      className="marginLeftAndRight"
    >
      Mathematics is not about numbers, equations, or algorithms: it is about understanding. —William Paul Thurston
    </p>
    <p
      className="marginLeftAndRight"
    >
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </p>
    <p
      className="marginLeftAndRight"
    >
      Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers. — Shakuntala Devi, Indian writer and mental calculator
    </p>
    <p
      className="marginLeftAndRight"
    >
      Mathematics is the most beautiful and most powerful creation of the human spirit. — Stefan Banach, Polish mathematician
    </p>
    <p
      className="marginLeftAndRight"
    >
      We will always have STEM with us. Some things will drop out of the public eye and go away, but there will always be science, engineering, and technology. And there will always, always be mathematics. — Katherine Johnson, African-American mathematician
    </p>
    <p
      className="marginLeftAndRight"
    >
      Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. — Richard Courant, German-American mathematician
    </p>
  </div>
</div>
`;
